selfdrive/car/gm/values.py

```diff
@@ -37,28 +38,42 @@ def __init__(self, CP):
     self.ZERO_GAS = 2048  # Coasting
     self.MAX_BRAKE = 400  # ~ -4.0 m/s^2 with regen
 
+    # The max amount of deceleration possible using ASCMGasRegenCmd alone. May vary by speed on some cars
+    self.MAX_REGEN_ACCEL_BP = [0.]  # m/s
+    self.MAX_REGEN_ACCEL_V = [0.]  # m/s^2
+
     if CP.carFingerprint in CAMERA_ACC_CAR:
+      # Camera ACC vehicles have no regen while enabled.
+      # Camera transitions to MAX_ACC_REGEN from ZERO_GAS and uses friction brakes instantly
       self.MAX_GAS = 3400
       self.MAX_ACC_REGEN = 1514
       self.INACTIVE_REGEN = 1554
-      # Camera ACC vehicles have no regen while enabled.
-      # Camera transitions to MAX_ACC_REGEN from ZERO_GAS and uses friction brakes instantly
-      max_regen_acceleration = 0.
 
     else:
       self.MAX_GAS = 3072  # Safety limit, not ACC max. Stock ACC >4096 from standstill.
       self.MAX_ACC_REGEN = 1404  # Max ACC regen is slightly less than max paddle regen
       self.INACTIVE_REGEN = 1404
-      # ICE has much less engine braking force compared to regen in EVs,
-      # lower threshold removes some braking deadzone
-      max_regen_acceleration = -1. if CP.carFingerprint in EV_CAR else -0.1
 
-    self.GAS_LOOKUP_BP = [max_regen_acceleration, 0., self.ACCEL_MAX]
-    self.GAS_LOOKUP_V = [self.MAX_ACC_REGEN, self.ZERO_GAS, self.MAX_GAS]
+      if CP.carFingerprint in EV_CAR:
+        # determined by letting Volt regen to a stop in L gear from 89mph,
+        # and by letting off gas and allowing car to creep, for determining
+        # the positive threshold values at very low speed
+        self.MAX_REGEN_ACCEL_BP = [0, 2.0, 10.]
```

Comment:
I think it should also have the CP at 20m/s, where the slope in the regen decal vs speed graph changes. Then the low speed regen acceleration value should be 0.1 rather than 0.5 in order to avoid rough stops that I commented about in response to your review comment @sshane